.github/workflows/nightly_test.yml

```diff
@@ -95,6 +101,21 @@ jobs:
     - name: Run test
       uses: ./.github/actions/orca-python-py37-spark3-action
 
+  Orca-Python-Horovod-Tf2-Py37-Spark3:
+    #if: ${{ github.event.schedule || github.event.inputs.artifact == 'Orca-Python-Horovod-Tf2-Py37-Spark3' || github.event.inputs.artifact == 'all' }}
+    runs-on: [self-hosted, Gondolin, ubuntu-20.04-lts]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK8
+      uses: ./.github/actions/jdk-setup-action
+    - name: Set up maven
+      uses: ./.github/actions/maven-setup-action 
+    - name: Setup env
+      uses: ./.github/actions/orca/setup-env/setup-horovod-tf2-py37
+    - name: Run Test
+      uses: ./.github/actions/orca/orca-python-horovod-tf2-py37-spark3-action/nightly-test
+      
```

Comment:
Will a remove-env action be added?

Reply:
removing the env will cause each test to take too long, will no longer add remove-env.